src/cmds/scripts/pbs_topologyinfo.py

```diff
@@ -48,20 +48,21 @@ def reportsockets_win(f):
     temp = f.read().split(',')
     for item in temp:
         if item.find('sockets:') != -1:
-            sockets = int(item.replace('sockets:', ''))
+            sockets = int(item[len('sockets:'):])
```

Comment:
I find this hard to read.  I'd rather you add a comment saying 'len('sockets:') = 8 and make the splice of item[8:] (and for the subsequent two splices).